packages/core/src/index.html

```diff
@@ -10,35 +10,107 @@
     <script nomodule src="/build/dso-toolkit.js"></script>
   </head>
   <body>
-    <dso-alert status="success">
-      Dit is een testmelding
-    </dso-alert>
-    <dso-alert status="info">
-      Dit is een testmelding
-    </dso-alert>
-    <dso-alert status="warning">
-      Dit is een testmelding
-    </dso-alert>
-    <dso-alert status="danger">
-      Dit is een testmelding
-    </dso-alert>
-    <dso-badge>
-      Testbadge
-    </dso-badge>
-    <dso-badge level="primary">
-      Testbadge
-    </dso-badge>
-    <dso-badge level="success">
-      Testbadge
-    </dso-badge>
-    <dso-badge level="info">
-      Testbadge
-    </dso-badge>
-    <dso-badge level="warning">
-      Testbadge
-    </dso-badge>
-    <dso-badge level="danger">
-      Testbadge
-    </dso-badge>
+    <div class="row">
```

Comment:
Ik begrijp waar je naar toe wilt, maar in een `.row` mag alleen een kolom definitie staan (`col-*`).

Volgens mij wil je de verschillende componenten onder elkaar groeperen. Een gewone `<div>` is dan al genoeg. Als verticale lucht een probleem is kan je dat met een `<br>` of `<hr>` oplossen